lms/djangoapps/grades/signals/handlers.py

```diff
@@ -109,6 +109,25 @@ def submissions_score_reset_handler(sender, **kwargs):  # pylint: disable=unused
     )
 
 
+@contextmanager
+def disconnect_submissions_signal_receiver(signal=score_set):
```

Comment:
Given the naming of this method I'd consider not setting a default value for `signal`. The usage of the contextmanager reads a little strangely without knowing what `signal` is. As someone new to this code my first question would be,  "Disconnect which submissions signal receiver?"